packages/teams-js/test/private/meetingRoom.spec.ts

```diff
@@ -114,32 +121,38 @@ describe('meetingRoom', () => {
       );
     });
 
+    it('sendCommandToPairedMeetingRoom should throw error when meetingRoom is not supported.', async () => {
+      await framelessPlatformMock.initializeWithContext('content');
```

Comment:
Changes look good, but just noticing that we are not testing these unit tests for all frame contexts.

Reply:
I approved the changes, so you don't get blocked on this

Reply:
Thanks